test/e2e/resize_nodes.go

```diff
@@ -560,6 +566,93 @@ var _ = Describe("Nodes [Disruptive]", func() {
 					}
 				}
 			})
+
+			// What happens in this test:
+			// 	Network connectivity on a node is cut off for 120 seconds to simulate network partition
+			// Expect to observe:
+			// 1. Node is marked NotReady after timeout by nodecontroller (40seconds)
+			// 2. All pods on node are marked NotReady shortly after #1
+			// 3. Node and pods return to Ready after connectivivty recovers
```

Comment:
I think there's a test that already does 1, 3 in a slightly less destructive way than dropping all traffic+sleep+undropping, can you just have it do 2 as well (https://github.com/kubernetes/kubernetes/blob/master/test/e2e/resize_nodes.go#L501)?